github/compliance/issue.py

```diff
@@ -316,6 +316,27 @@ def labels_to_preserve():
         raise RuntimeError('this line should never be reached') # all cases should be handled before
 
 
+def close_issue_with_retry(
+    issue: github3.issues.ShortIssue,
+) -> bool:
+    '''
+    close issue with recursive retries
+    '''
+    try:
+        return issue.close()
+
+    except github3.exceptions.UnprocessableEntity:
+        # assignee was suspended from github
```

Comment:
do we know this for sure?  if it is just _one_ (likely) cause, we should say so

Reply:
will re-phrase 👍🏻